rudder-web/src/main/scala/com/normation/rudder/web/components/TagsEditForm.scala

```diff
@@ -0,0 +1,44 @@
+package com.normation.rudder.web.components
+
+import net.liftweb.http.DispatchSnippet
+import net.liftweb.http.js._
+import JsCmds._
+import JE._
+import com.normation.rudder.domain.policies.Tags
+
+
+class TagsEditForm(
+  tags : JsObj
+) extends DispatchSnippet {
+
+  def dispatch = {
+    case "cfTagsDirectiveConfiguration" => (xml) => cfTagsConfiguration("tagsDirectiveCtrl")
+    case "cfTagsRuleConfiguration"      => (xml) => cfTagsConfiguration("tagsRuleCtrl")
+  }
+
+  var listTags = tags
+  def cfTagsConfiguration(controllerID:String) =
+    <div class="row form-group" ng-controller={{controllerID}} ng-cloak="" >
```

Comment:
Would be better if html was in an html file and only modifying the ng-controller attribute using lift css selector (like we do for directive inputs"